### ISSUE TITLE
Raccoons can eat... air

### ISSUE THREAD

**Issue by TBiscuit1** [Alex-the-666/AlexsMobs#462]
So don't ask me why I did that, but I wanted to try what happen when you make Air edible, then, I heard a very VERY loud eating sound just to see that a raccoon was EATING the air XD

Used Crafttweaker to make Air edible
Script in question
`import crafttweaker.api.food.MCFood;`
`var Air = <item:minecraft:air>;`
`Air.food = new MCFood(2, 0.6);`

Reddit Post that I made cause it was funny
https://www.reddit.com/r/PhoenixSC/comments/nopnvh/accidentally_made_air_edible_while_making_a/

**Comment by TBiscuit1** [comment]
Oh and btw, it's so funny, if you can let it go as an easter egg (kind of) I would be really happy XD

**Comment by Kushagra-Srivastava** [comment]
it would be tasty ummmm...

**Comment by AidanCorgi4556** [comment]
This is not a glitch this is something you have made possible on your own???

**Comment by TBiscuit1** [comment]
> This is not a glitch this is something you have made possible on your own???

Yes, but the problem is that the racoon eats it instantly and create a very VERY loud noise

**Comment by AidanCorgi4556** [comment]
And why should this be fixed? Youre the only person who will ever do this and theres no use to making racoons eat air. Its probably something thats very hard to change.

**Comment by Kushagra-Srivastava** [comment]
_`laughs loudly`_
